CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Unreleased Changes
+
+**Breaking Changes**
+
+### Constructs upgrade (`v3` to `v10`)
+
+The `constructs` package serves as the base for all building blocks of the CDKs (e.g. CDK for Terraform, AWS CDK and cdk8s). Recently the `constructs` package was lifted to v10 with a few but major changes. The reason behind those was to make it future proof and stable across different CDKs. In this process the API surface of the `constructs` base package was removed and some functionality has been moved directly to the AWS CDK instead. For more information and the underlying motivation for the changes to the `constructs` library refer to the [Removal of Construct Compatibility Layer RFC](https://github.com/aws/aws-cdk-rfcs/blob/master/text/0192-remove-constructs-compat.md).
```

Comment:
> In this process the API surface of the `constructs` base package was removed and some functionality has been moved directly to the AWS CDK instead

This doesn't sound like proper english to me, but as it's also not my first language I'd like to summon @danieldreier to keep us grammatically correct :D

Reply:
Yeah, you're right :D And I probably meant "was reduced" instead of "was removed" 😅